### PR TITLE
[terra-tab] Changed blue in tab hyperlinks to pass WCAG.

### LINKED ISSUE #841: Accessibility issue in Terra-tabs
# Issue Description
In the latest version of terra-tabs, we can see few accessibility violations when it is inspected [here](https://engineering.cerner.com/terra-framework/raw/tests/terra-tabs/tabs/tabs/controlled-tabs) 
The same violations are seen while running WDIO test for accessibility

## Additional Context / Screenshots
image

### PR DESCRIPTION
**Pull request by jeremyfuksa**
Signed-off-by: Jeremy Fuksa <hello@orangefla.me>

### Summary
Closes #841. Changed blue in hyperlinks to pass WCAG testing.

Please add an entry in the CHANGELOG.md file in each package you modify.

Thanks for contributing to Terra.
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
These likely need screenshot update as well

**Comment by benbcai** [comment]
@jeremyfuksa Can you add an entry to the [terra-tab Changelog](https://github.com/cerner/terra-framework/blob/master/packages/terra-tabs/CHANGELOG.md) and regenerate any screenshots to resolve the build failure?

File: packages/terra-tabs/src/ModularTabs.module.scss [hunk 1]
```diff
@@ -57,11 +57,11 @@
```
Comment (jmsv6d) [inline packages/terra-tabs/src/ModularTabs.module.scss]: Changes to the modular tabs scss file isn't necessary, it's dead code.
As a side note: Weren't we going to delete the modular tab code?

Comment (ryanthemanuel) [reply packages/terra-tabs/src/ModularTabs.module.scss]: Yeah we can do that, but it can be done on a later pull request.

**Pull request merged**